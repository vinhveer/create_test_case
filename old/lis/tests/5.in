81
914 121 704 522 965 354 479 44 436 29 245 672 130 466 999 17 571 451 592 854 206 798 749 848 345 44 42 761 64 282 360 129 135 587 391 836 620 801 138 482 770 721 927 170 892 538 607 919 704 860 111 39 532 560 701 289 630 337 218 296 700 465 23 881 63 711 645 153 677 724 378 716 47 266 959 817 801 853 696 395 405 
55
223 887 630 919 206 849 137 668 287 895 235 255 416 107 806 451 41 186 499 207 548 587 255 690 879 549 592 872 437 466 656 457 861 430 483 561 173 223 22 446 287 216 621 750 441 852 352 436 610 334 819 337 268 596 162 
