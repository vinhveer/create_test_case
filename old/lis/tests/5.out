2
206 587 
